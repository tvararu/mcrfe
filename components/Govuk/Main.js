export default ({ children }) => (
  <main className="govuk-main-wrapper" id="main-content" role="main">
    {children}
  </main>
);
