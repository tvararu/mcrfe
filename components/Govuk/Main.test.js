import renderer from "react-test-renderer";
import Main from "./Main";

describe("Main", () => {
  it("renders correctly", () => {
    const component = renderer.create(<Main>Lorem</Main>);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
