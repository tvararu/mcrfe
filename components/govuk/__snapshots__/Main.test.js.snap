// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main renders correctly 1`] = `
<main
  className="govuk-main-wrapper"
  id="main-content"
  role="main"
>
  Lorem
</main>
`;
